tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -85,6 +85,7 @@ static const struct device *const devices[] = {
 	DEVS_FOR_DT_COMPAT(nxp_lpc_rtc)
 #endif
 #ifdef CONFIG_TEST_DRIVER_COUNTER_MCUX_LPC_RTC_HIGHRES
+#define COUNTER_HAS_INACCURACY
```

Comment:
What about something like
#define COUNTER_ACCEPTED_ACCURACY 10 //and 30 for NXP LPC